src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -196,6 +197,26 @@ public final void onCompleted(final Run run, @Nonnull final TaskListener listene
           statsd = getDescriptor().leaseClient();
           statsd.incrementCounter("completed", tagsToCounter);
           logger.fine(String.format("Attempted to send 'completed' counter with tags: %s", Arrays.toString(tagsToCounter)));
+
+          // Send KPIs 
+          if (!isMasterBranch(run)) {
+            if (run.getResult() == Result.SUCCESS) {
+              long mttr = getMeanTimeToRecovery(run);
+              long cycleTime = getCycleTime(run);;
+              long leadTime = run.getDuration() + mttr;
+
+              statsd.gauge("cycletime", cycleTime / 1000, tagsToCounter);
+              statsd.gauge("leadtime", leadTime / 1000, tagsToCounter);
+              statsd.gauge("mttr", mttr / 1000, tagsToCounter);
+            } else {
+              long feedbackTime = run.getDuration();
+              long mtbf = getMeanTimeBetweenFailure(run);
+
+              statsd.gauge("feedbacktime", feedbackTime / 1000, tagsToCounter);
+              statsd.gauge("mtbf", mtbf / 1000, tagsToCounter);
```

Comment:
Submitting `0` values could lead to incorrect aggregation over various timeframes. In these cases, we should only submit the metric if the value `!= 0`. Looks like this could be the case for `mttr`, `cycleTime`, and `mtbf`.

Reply:
Yes, that makes sense. Will change it.